spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/PubSubAdmin.java

```diff
@@ -136,6 +140,8 @@ public void deleteTopic(String topicName) {
 	 * Return every topic in a project.
 	 *
 	 * <p>If there are multiple pages, they will all be merged into the same result.
+	 *
```

Comment:
no newline because it's a method. While you are there, can you remove the newline before <p>?